setup/symfony_server.rst

```diff
@@ -57,6 +57,22 @@ run the Symfony server in the background:
     # show the latest log messages
     $ symfony server:log
 
+Enabling PHP-FPM
+----------------
+
+When the server starts it will check the ``public`` folder for an ``index.php``
+file. If this file is found the server will automatically start with PHP-FPM
```

Comment:
@fabpot can you please check the first sentence? From your comment in the referenced issue, I get the feeling that more front-controller files are checked than just `public/index.php`?

Reply:
Yes, it checks for "common" patterns like `web/app.php`, `web/app_dev.php` or `public/index.php`.

Reply:
I have changed the text to mention those common patterns